test/mpi/rma/fence_shm.c

```diff
@@ -59,6 +59,12 @@ int main(int argc, char *argv[])
             MPI_Win_fence(0, shm_win);
         }
 
+        /* Other ranks simply join fence */
+        if (shm_rank > 1) {
+            MPI_Win_fence(0, shm_win);
+            MPI_Win_fence(0, shm_win);
```

Comment:
Comments: why need two win_fence calls here?

Reply:
Because rank 0 calls two fences, thus rank > 1 has to also call fence twice to match the synchronization.

Reply:
oh, didn't see un-expanded  code before. thanks.